torch/testing/_internal/common_methods_invocations.py

```diff
@@ -546,6 +557,7 @@ def __init__(self,
                  test_conjugated_samples=True,
                  test_neg_view=True,
                  assert_jit_shape_analysis=False,  # assert that jit shape analysis fully propagates shape
+                 normal_filter=(lambda _: torch.tensor(False), 0)  # Filter for singular input values
```

Comment:
name bikeshedding: "reference_numerics_filter"?

We can probably change this to "None".

This should be added to the UnaryUfuncInfo class, not the base OpInfo class, because the only test that uses it is applied exclusively to UnaryUfuncInfos.